webapp/sources/rudder/rudder-web/src/main/elm/sources/Tags/Init.elm

```diff
@@ -0,0 +1,26 @@
+port module Tags.Init exposing (..)
+
+import Json.Decode exposing (..)
+
+import Tags.DataTypes exposing (..)
+import Tags.JsonEncoder exposing (..)
+
+
+-- PORTS / SUBSCRIPTIONS
+port updateResult : String -> Cmd msg
+port addToFilters : Value  -> Cmd msg
+
+subscriptions : Model -> Sub Msg
+subscriptions model =
+  Sub.none
+
+init : { contextPath : String, hasWriteRights : Bool, tags : List Tag, filterId : String, isEditForm : Bool, isRule : Bool, objectId : String } -> ( Model, Cmd Msg )
+init flags =
+  let
+    objectType = if flags.isRule then "rule" else "directive"
```

Comment:
I'd rather have `init` take an objectType string directly instead of a boolean if it's not too much work